CoMD/src-threaded/CMakeLists.txt

```diff
@@ -73,6 +77,18 @@ target_compile_options(
   coMD_threaded
   PUBLIC
   ${comd_compile_options})
+elseif(ENABLE_HIP)
+  include_directories(
```

Comment:
Are these all really required? If so, they should be `target_include_directories` specific to the targets that require them.

Reply:
Thanks, I was duplicating this logic from the CUDA and MPI versions of this. I'll look in to making this a bit better. It's odd that the `CXXFLAGS` are added.